app/invocation/invocation.css

```diff
@@ -1035,7 +1035,7 @@
   margin-bottom: 2px;
 }
 
-.metadata-title:not(:first-child) {
+.metadata-title:not(:first-child),  .snapshot-id-container {
```

Comment:
This formatting looks a little off (two spaces after `,`) - looks like `prettier.sh` was broken. Mind re-running buildfix once https://github.com/buildbuddy-io/buildbuddy/pull/8303 is in?

Reply:
Done